src/DSA/Trie/Trie_test.go

```diff
@@ -0,0 +1,31 @@
+package trie
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTrie(t *testing.T) {
+	trie := NewTrie()
+
+	trie.Insert("foo")
+	trie.Insert("fool")
+	trie.Insert("foolish")
+	trie.Insert("bar")
+
+	result := trie.Find("fo")
```

Comment:
ThePrimeagen also sort the result, maybe we also need that https://github.com/ThePrimeagen/kata-machine/blob/master/src/__tests__/Trie.ts#L10